pkg/rc/replication_controller.go

```diff
@@ -346,6 +346,28 @@ func (rc *replicationController) meetDesires(rcFields fields.RC) error {
 		}
 	}
 
+	// check if a node transfer node has been removed out of band
+	status, _, err := rc.rcStatusStore.Get(rc.rcID)
+	if err != nil && !statusstore.IsNoStatus(err) {
+		rc.logger.WithError(err).Errorln("Unable to fetch RC status to check for in progress node transfer")
+		return err
+	} else if !statusstore.IsNoStatus(err) {
+		hasOldNode := false
+		for _, node := range current.Nodes() {
+			if status.NodeTransfer.OldNode == node {
+				hasOldNode = true
+			}
+		}
+		if !hasOldNode {
```

Comment:
In what cases does the old node get removed during a node transfer?

Reply:
A p2 operator may want to forcefully remove a node while a transfer was in progress (perhaps it didn't finish in time or the replica on the new node was not coming up healthy). Currently, the operator would have to be sure to clean up things like labels manually but with this code change that will be taken care off.